dts/bindings/sensor/sensor-device.yaml

```diff
@@ -17,3 +17,18 @@ properties:
       This property is defined in the Generic Sensor Property Usages of the HID
       Usage Tables specification
       (https://usb.org/sites/default/files/hut1_3_0.pdf, section 22.5).
+
+  reporters:
+    type: phandle-array
+    description: |
+      Sensor input reporter specifiers, used by virtual sensors to synthesize
+      sensor channels from one or more other sensors. For example, a lid angle
+      sensor synthesized from a lid accelerometer and a base accelerometer.
+
+  reporter-names:
+    type: string-array
+    description: Provided names of sensor input reporter specifiers
+
+reporter-cells:
+  - interval
+  - sensitivity
```

Comment:
What's the policy for documenting these "cell" fields.  I don't find any current examples in any other yaml files (gpio-cells, io-cells, etc).

Reply:
I'm not sure these should be here, what if we have 2 virtual sensors with the same reporter but different `interval` and `sensitivity`? I'm assuming that these are used to configure the reported during init?

Reply:
@keith-zephyr I don't think we have a general convention for documenting cells.

@yperess These are initial values that the virtual sensor client will request from the reporter during boot. The sensor subsystem will arbitrate if multiple virtual sensors request different `interval` or `sensitivity` values from the same reporter.

Reply:
But these values are being added in the driver level and not as a part of the subsystem (which runs in userspace). So I'm not sure I'm following since the `init()` function of these drivers is at a layer below the sensor subsystem and will be requesting the configuration directly from the drivers no? In pseudo code, what I'm envisioning this driver implementation doing is:

```
static int my_sensor_init(const struct device *dev)
{
  // Loop through the reporters
  for (reporter in dev->reporters) {
    // Request the configs
    sensor_config(reporter.config)
  }
}
```

Since this is at the kernel driver level we can't call into the subsystem since some people might not want to use it. I suppose what I'm thinking is that this should go outside of `sensor-device.yaml` and effectively be a subclass `virtual-sensor-device.yaml` and some dependency on a Kconfig like `CONFIG_SENSOR_SUBSYSTEM`.